pallets/thea-executor/src/lib.rs

```diff
@@ -470,6 +471,23 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(7)]
+		#[pallet::weight(< T as Config >::TheaExecWeightInfo::claim_deposit(1))]
+		#[transactional]
+		pub fn create_parachain_asset(
```

Comment:
Can we also take the information like token name and token ticker here as well, otherwise we would need add one more process to the listing process where the governance need to call pallet_assets::set_matadata() to make it show up properly in the assets tab. also once its made consistent with an event, the FE can index this for automatically updating information on indexers.

Reply:
I dont think it's reqd, set_metadata can batched together with this call when raising proposal. The objective of this to remove req to transfer token.

Reply:
it cant be batched, because we only know the assetid after this has been called.

Reply:
resolved